### PR TITLE
Edit functionality added for user

### PR DESCRIPTION
**Pull request by abdullokhtolibjonov**

### CODE REVIEW

File: Parking-Zone/Areas/Identity/Pages/Account/Manage/Index.cshtml.cs [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            [Display(Name = "Phone number")]
+            public string PhoneNumber { get; set; }
+        }
+
+        private async Task LoadAsync(AppUser user)
+        {
+            var userName = await _userManager.GetUserNameAsync(user);
+            var phoneNumber = await _userManager.GetPhoneNumberAsync(user);
+
+            Username = userName;
```
Comment (DilshodMQ) [inline Parking-Zone/Areas/Identity/Pages/Account/Manage/Index.cshtml.cs]: why do you need this UserName

File: Parking-Zone/Areas/Identity/Pages/Account/Manage/Index.cshtml.cs [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            if (Input.PhoneNumber != phoneNumber)
+            {
+                var setPhoneResult = await _userManager.SetPhoneNumberAsync(user, Input.PhoneNumber);
+                if (!setPhoneResult.Succeeded)
+                {
+                    StatusMessage = "Unexpected error when trying to set phone number.";
+                    return RedirectToPage();
+                }
+            }
+
```
Comment (DilshodMQ) [inline Parking-Zone/Areas/Identity/Pages/Account/Manage/Index.cshtml.cs:115]: where is the FullName change code?

File: Parking-Zone/Areas/Identity/Pages/Account/Manage/_Layout.cshtml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+<h1>Manage your account</h1>
+
+<div>
+    <h2>Change your account settings</h2>
+    <hr />
+    <div class="row">
+        <div class="col-md-3">
+            <partial name="_ManageNav" />
+        </div>
```
Comment (DilshodMQ) [inline Parking-Zone/Areas/Identity/Pages/Account/Manage/_Layout.cshtml]: we don't need this partial

File: Parking-Zone/Areas/Identity/Pages/Account/Manage/_ManageNav.cshtml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    <li class="nav-item"><a class="nav-link @ManageNavPages.IndexNavClass(ViewContext)" id="profile" asp-page="./Index">Profile</a></li>
+    <li class="nav-item"><a class="nav-link @ManageNavPages.EmailNavClass(ViewContext)" id="email" asp-page="./Email">Email</a></li>
+    <li class="nav-item"><a class="nav-link @ManageNavPages.ChangePasswordNavClass(ViewContext)" id="change-password" asp-page="./ChangePassword">Password</a></li>
+    @if (hasExternalLogins)
+    {
+        <li id="external-logins" class="nav-item"><a id="external-login" class="nav-link @ManageNavPages.ExternalLoginsNavClass(ViewContext)" asp-page="./ExternalLogins">External logins</a></li>
+    }
+    <li class="nav-item"><a class="nav-link @ManageNavPages.TwoFactorAuthenticationNavClass(ViewContext)" id="two-factor" asp-page="./TwoFactorAuthentication">Two-factor authentication</a></li>
+    <li class="nav-item"><a class="nav-link @ManageNavPages.PersonalDataNavClass(ViewContext)" id="personal-data" asp-page="./PersonalData">Personal data</a></li>
+</ul>
```
Comment (DilshodMQ) [inline Parking-Zone/Areas/Identity/Pages/Account/Manage/_ManageNav.cshtml]: why do you need this page?

**Review by DilshodMQ — CHANGES_REQUESTED** [review]

File: Parking-Zone/Areas/Identity/Pages/Account/Register.cshtml.cs [hunk 5]
```diff
@@ -97,6 +98,17 @@ public class InputModel
             [Display(Name = "Confirm password")]
             [Compare("Password", ErrorMessage = "The password and confirmation password do not match.")]
             public string ConfirmPassword { get; set; }
+
+            [Required]
+            [DataType(DataType.Text)]
+            [Display(Name = "Full Name")]
+            public string FullName { get; set; }
+
+
```
Comment (napster1991) [inline Parking-Zone/Areas/Identity/Pages/Account/Register.cshtml.cs]: Remove this line

File: Parking-Zone/appsettings.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Server=DESKTOP-U8CO9CB;Database=ParkingZone3;Trusted_Connection=True;TrustServerCertificate=True;"
+    "DefaultConnection": "Server=DESKTOP-U8CO9CB;Database=ParkingZone4;Trusted_Connection=True;TrustServerCertificate=True;"
```
Comment (napster1991) [inline Parking-Zone/appsettings.json:3]: Why are you constantly switching from one db to another?

**Review by napster1991 — CHANGES_REQUESTED** [review]

**Review by napster1991 — APPROVED** [review]

**Review by DilshodMQ — APPROVED** [review]

**Pull request merged**